### PR TITLE
New Features about `Number.prototype.toLocaleString()` for IE11 in win7 and 8, a lot of related new definitions

### PR DESCRIPTION
**Pull request by Clayblockunova**
I think `useLocaleNumeralDefinition()` should also be updated to ensure all paddings included.

### CODE REVIEW

**Comment by Clayblockunova** [comment]
can I remove all `-Infinity` references?

File: src/lib/features.js [hunk 1]
```diff
@@ -552,6 +552,47 @@ var featureInfos =
...
+        'the Polish string representation of Infinity ("+nieskończoność"), the letters in the ' +
+        'Russian string representation of Infinity ("бесконечность"), the characters in the ' +
+        'Chinese string representation of Infinity ("正无穷大" or "正無窮大"), the characters in ' +
+        'the Simplified Chinese string representation of NaN ("非数字").',
+        check:
+        function ()
+        {
+            var available =
+            checkLocaleNumeral('ar', NaN, /^ليس برقم/) &&
+            checkLocaleNumeral('ar', Infinity, /^+لا.نهاية/) &&
```
Comment (fasttime) [inline src/lib/features.js:578]: I'm getting a syntax error on this regular expression.

**Comment by fasttime** [comment]
> can I remove all `-Infinity` references?

If the tests pass I'd say yes.

File: src/lib/features.js [hunk 2]
```diff
@@ -552,6 +552,47 @@ var featureInfos =
...
+        'Chinese string representation of Infinity ("正无穷大" or "正無窮大"), the characters in ' +
+        'the Simplified Chinese string representation of NaN ("非数字").',
+        check:
+        function ()
+        {
+            var available =
+            checkLocaleNumeral('ar', NaN, /^ليس.برقم/) &&
+            checkLocaleNumeral('ar', Infinity, /^\+لا.نهاية/) &&
+            checkLocaleNumeral('ar-td', 234567890.1, /^٢٣٤٬?٥٦٧٬?٨٩٠٫١/) &&
+            checkLocaleNumeral('cz', NaN, /^Není.číslo/) &&
```
Comment (fasttime) [inline src/lib/features.js:580]: I'm trying this in IE 11 on Windows 7, but `NaN.toLocaleString('cz')` results in `'NaN'`.

File: src/lib/features.js [hunk 3]
```diff
@@ -552,6 +552,47 @@ var featureInfos =
...
+            checkLocaleNumeral('el', Infinity, /^Άπειρο/) &&
+            checkLocaleNumeral('el', NaN, /^μη.αριθμός/) &&
+            checkLocaleNumeral('he', NaN, /^לא.מספר/) &&
+            checkLocaleNumeral('ja', Infinity, /^\+∞/) &&
+            checkLocaleNumeral('ja', NaN, /^NaN.(非数値)/) &&
```
Comment (fasttime) [inline src/lib/features.js:585]:
```suggestion
            checkLocaleNumeral('ja', NaN, /^NaN \(非数値\)/) &&
```

File: src/lib/features.js [hunk 4]
```diff
@@ -552,6 +552,47 @@ var featureInfos =
...
+            checkLocaleNumeral('lt', Infinity, /^begalybė/) &&
+            checkLocaleNumeral('lv', Infinity, /^bezgalība/) &&
+            checkLocaleNumeral('pl', Infinity, /^\+nieskończoność/) &&
+            checkLocaleNumeral('ru', Infinity, /^бесконечность/) &&
```
Comment (fasttime) [inline src/lib/features.js:589]: `Infinity.toLocaleString('lv')` and `Infinity.toLocaleString('ru')` are giving `"∞"` on Windows 8.

Comment (Clayblockunova) [reply src/lib/features.js:589]: does `LOCALE_INFINITY` work on Windows 8?

**Comment by Clayblockunova** [comment]
what should I do now?

File: src/lib/definitions.js [hunk 5]
```diff
@@ -383,6 +383,9 @@ function getFHPaddingEntries(index)
     var LOCALE_INFINITY                 = Feature.LOCALE_INFINITY;
     var LOCALE_NUMERALS                 = Feature.LOCALE_NUMERALS;
     var LOCALE_NUMERALS_EXT             = Feature.LOCALE_NUMERALS_EXT;
+    var LOCALE_NUMERALS_IE11_WIN7       = Feature.LOCALE_NUMERALS_IE11_WIN7;
+    var LOCALE_NUMERALS_IE11_WIN7_8     = Feature.LOCALE_NUMERALS_IE11_WIN7_8;
+    var LOCALE_NUMERALS_IE11_WIN8       = Feature.LOCALE_NUMERALS_IE11_WIN8;
```
Comment (fasttime) [inline src/lib/definitions.js:388]: I'm not sure if this division makes sense. It seems that these features depend on the version of certain DLLs used by IE, and these DLLs could be updated independently from the operating system, for example with Windows updates. This is especially true for Windows 7 and 8 where IE was integrated with other system components.

Comment (Clayblockunova) [reply src/lib/definitions.js:388]: so, what should I do? should I just remove Latvian and Russian related definition?

Comment (fasttime) [reply src/lib/definitions.js:388]: I don't know yet. I will have to do some more analysis.

Comment (Clayblockunova) [reply src/lib/definitions.js:388]: is your analysis finished now?

Comment (Clayblockunova) [reply src/lib/definitions.js:388]: and, what should I do to Latvian and Russian related definition?

**Comment by Clayblockunova** [comment]
Sorry for closing this PR since I have something else to add.

**Pull request closed without merging**